cpu/stm32_common/periph/pm.c

```diff
@@ -49,13 +61,21 @@ void pm_set(unsigned mode)
             /* Clear PDDS and LPDS bits to enter stop mode on */
             /* deepsleep with voltage regulator on */
             PWR->CR &= ~(PWR_CR_PDDS | PWR_CR_LPDS);
+            PWR->CR |= PM_STOP_CONFIG;
             /* Set SLEEPDEEP bit of system control block */
             deep = 1;
             break;
     }
 #endif
 
     cortexm_sleep(deep);
+
+#if defined(CPU_FAM_STM32F1) || defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4)
```

Comment:
Interrupts are disabled in [pm_set_lowest](https://github.com/RIOT-OS/RIOT/blob/master/sys/pm_layered/pm.c#L61) so I guess interrupts are only handled after executing the clock init code and re-enabling interrupts in pm_set_lowest